fe/src/main/java/org/apache/doris/catalog/CatalogRecycleBin.java

```diff
@@ -277,6 +284,12 @@ public synchronized void replayEraseTable(long tableId) {
             }
         }
 
+        // colocation
+        if (Catalog.getCurrentColocateIndex().removeTable(tableId)) {
+            ColocatePersistInfo colocateInfo = ColocatePersistInfo.createForRemoveTable(tableId);
```

Comment:
when we replay erase table? Why we need to logColocateRemoveTable. I think when this log had been logged, the logColocateRemoveTable had been logged too

Reply:
Table will be erased after trash expired.
Using 2 edit logs is a problem left over by history...I will try to merge these 2 logs

Reply:
Forget it.... it will change a lot....